tests/MongoDB.Analyzer.Tests.Common.TestCases/Linq/NotSupportedLinqExpressions.cs

```diff
@@ -130,5 +130,15 @@ public void Query_syntax()
                 where ReturnArgument(ReturnArgument(ReturnArgument(this).ReturnArgument(person.Vehicle.VehicleType))).Type == VehicleTypeEnum.Bus
                 select person;
         }
+
+        [InvalidLinq("Convert(Convert((Convert({document}{EnumInt16}, Int32) + 10), GenType_Enum_12), Int32) is not supported.")]
+        [InvalidLinq3("Cannot find serializer for Convert(u.EnumInt16, Int32).", version: "[2.14.0-beta1, 2.18.0)")]
```

Comment:
Let's have constants for versions (DriverVersions)

Reply:
Done